lesson20/index.js

```diff
@@ -0,0 +1,113 @@
+
+const body = document.querySelector("body");
+
+const createElementWithClassName = (type, className) => {
+  const element = document.createElement(type);
+  element.className = className;
+  return element;
+};
+
+const createLoading = () => {
+  const loading = createElementWithClassName("img", "loading");
+  loading.id = "js-loading";
+  loading.src = "./loading-circle.gif";
+  return loading;
+};
+
+const removeLoading = () => document.getElementById("js-loading").remove();
```

Comment:
【thoughts】Looking at the arrow function this time, I thought it was simple and nice. I would like to imitate it in the future.

Reply:
Thanks for the review :)